### PR TITLE
Unbreak build on FreeBSD

### PR DESCRIPTION
**Pull request by t6**

### CODE REVIEW

File: src/afl-forkserver.c [hunk 1]
```diff
@@ -40,12 +40,18 @@
 #include <sys/wait.h>
 #include <sys/resource.h>
 
+#if defined(__APPLE__) || defined(__FreeBSD__) || defined(__OpenBSD__)
+#define HAVE_ARC4RANDOM 1
+#endif                           /* __APPLE__ || __FreeBSD__ || __OpenBSD__ */
+
```
Comment (t6) [inline src/afl-forkserver.c]: Should HAVE_ARC4RANDOM maybe be moved to `include/config.h` instead?

Comment (vanhauser-thc) [reply src/afl-forkserver.c]: in my repository state it is in ./include/afl-fuzz.h which is fine

Comment (vanhauser-thc) [reply src/afl-forkserver.c]: ah sorry, now I get it. yes I will move it.

Comment (vanhauser-thc) [reply src/afl-forkserver.c]: will move it to config.h

**Comment by vanhauser-thc** [comment]
OK made the changes, and fixed a compile break you introduced for non-BSD :)

please check that everything still works

**Comment by t6** [comment]
> OK made the changes, and fixed a compile break you introduced for non-BSD :)

Oops!

> please check that everything still works

Yes, everything still works. Thank you!

**Pull request merged**